pkg/apis/wordpress/v1alpha1/wordpress_types.go

```diff
@@ -117,6 +117,14 @@ type WordpressSpec struct {
 	// PodMetadata allow setting custom labels/annotations on wordpress pods
 	// +optional
 	PodMetadata *metav1.ObjectMeta `json:"podMetadata,omitempty"`
+	// ReadinessProbe allows setting a custom readiness probe for the wordpress container.
+	// If not specified, a default probe that makes a HTTP request on the "/" path will be used.
+	// +optional
+	ReadinessProbe *corev1.Probe `json:"readinessProbe,omitempty"`
```

Comment:
What do you think about prfixing them with `Wordpress`?

```suggestion
	WordpressReadinessProbe *corev1.Probe `json:"readinessProbe,omitempty"`
```

Reply:
`Resources` is also only used for the `Wordpress` container, so I don't know...